### DISCUSSION TITLE
connection time out

### DISCUSSION THREAD

**Discussion by yuchengliu0519** [bol-edu/HLS-SOC-Discussions#59] [Q&A] [answered]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/47782190/d636c90d-c34b-42a8-bc66-32eef79ec666)
嗨，大家好，上圖是我在lab2的第二部份FIR with interface streaming遇到的問題，我再跑jupiter的時候，出現了connection time out，首先我確定板子是對的，bit跟hwh檔案都有拿正確且位置都有找到，然後上網查是說可能是dma的問題，附圖是我dma的連接方法，
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/47782190/eee907a7-714d-4a05-9dfb-dbdd13ea3106)
想請教各位有人也遇到這個問題嗎?請問是怎麼解決的

**Comment by kevinjantw** [comment]
你是用哪個租用板子呢？
試過換租用板子？

**Reply by yuchengliu0519** [reply]
您好:
    我用的是pynq_03的板子，我後來用pynq_06的板子就沒有問題了，謝謝您的幫忙

**Comment by emma16163** [comment]
你似乎沒有按Run Connection Automation，按他兩次後再手動接線，FIR_N11_STRM 的 pstrminput 接到 axi_dma_in_O 的 M_AXIS_MM2S，
FIR_N11 _STRM 的 pstrmoutput 接到axi_dma_out_O 的S_AXIS_S2MM(和你圖上一樣)
最終完成的Block Diagram應該長得如下
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/75108089/d6619b0b-b3e4-49bf-80cd-c2c5ebb8a89a)

**Reply by yuchengliu0519** [reply]
抱歉我說明沒清楚，我其實有按Run Connection Automation，我那張圖只是為了給各位看一下我在Run Connection Automation之前有先手動設定dma並接線，只是因為我懷疑我dma接的有問題才特別截接線這一段給各位看的，謝謝您的回覆

**Comment by boledulab** [comment, marked as answer]
PYNQ-Z2 板子的網路卡是有這種問題。過去同學們也有反應過。 KV260 好像沒有發生過這樣的問題。

**Comment by v00000v** [comment]
我原本也有遇到同樣的問題
後來換一塊板子就好了~
